packages/typespec-test/test/schemaRegistry/generated/typespec-ts/review/schema-registry.api.md

```diff
@@ -75,12 +75,12 @@ export interface SchemaOperationsListSchemaVersionsOptionalParams extends Operat
 
 // @public
 export interface SchemaOperationsOperations {
-    getSchemaById: (id: string, options?: SchemaOperationsGetSchemaByIdOptionalParams) => Promise<Uint8Array>;
-    getSchemaByVersion: (groupName: string, name: string, schemaVersion: number, options?: SchemaOperationsGetSchemaByVersionOptionalParams) => Promise<Uint8Array>;
+    getSchemaById: (id: string, accept: string, options?: SchemaOperationsGetSchemaByIdOptionalParams) => Promise<Uint8Array>;
```

Comment:
accept as parameter?

Reply:
@MaryGao could you help investigate why the accept is not being filtered by us if it's generate from tcgc as a constant type ?

Reply:
This is introduced by tcgc when the responses have more than one content type, the relevant `accept` type would not be a constant, but a `string` type.

Reply:
See revised PR change summary point 3: contentType/accept header parameter logic change, @joheredi let me know if you have any concerns about it? Thank you.